ChangeLog

```diff
@@ -1,3 +1,8 @@
+2022-02-07  David Declerck <david.declerck@ocamlpro.com>
```

Comment:
There are ChangeLog files in most directories, please move the entries that are not related to the build system (root) to the files in the matching directories.

Exception: commonly we don't add extra entries in tests/Changelog when adjusting/adding a test (only for bigger changes to the testsuite or helper files).

Reply:
Indeed, hadn't noticed. I'll make the change to all my PRs.